src/openapi/export/export.yaml

```diff
@@ -0,0 +1,61 @@
+openapi: '3.0.1'
+info:
+  title: Export definitions
+  version: 1.0.0
+components:
+  schemas:
+    PolygonalGeometry:
+      description: GeoJSON Polygonal Geometry
+      discriminator:
+        propertyName: type
+      type: object
+      oneOf:
+        - $ref: '../core/geojson.yaml#/components/schemas/Polygon'
+        - $ref: '../core/geojson.yaml#/components/schemas/MultiPolygon'
+
+    RoiFeatureProperties:
+      type: object
+      description: Properties of a Region of Interest (ROI) feature
+      properties:
+        maxResolutionDeg:
+          description: Maximum resolution in degrees
+          type: number
+          minimum: 3.35276126861572e-7
+          maximum: 0.703125
+          format: double
+        minResolutionDeg:
+          description: Minimum resolution in degrees
+          type: number
+          minimum: 3.35276126861572e-7
```

Comment:
change to resolution that support zoom level 22:
https://github.com/MapColonies/mc-utils/blob/7e5ef38f03b698dfda4c090614d3197b887aa2f1/src/geo/tiles.ts#L104